.gitignore

```diff
@@ -66,4 +66,10 @@ yarn-error.log
 /yarn-error.log
 yarn-debug.log*
 .yarn-integrity
-.rake_tasks~
\ No newline at end of file
+.rake_tasks~
+/public/packs
+/public/packs-test
+/node_modules
+/yarn-error.log
+yarn-debug.log*
+.yarn-integrity
```

Comment:
Hi @billymoroney1, can you remove this file, `application.js`, `development.rb`, `yarn.lock` and `package.json`? Those files are not needed in this PR. Thanks!

Reply:
Also, can you add the issue number like this `Fixes #0000`? Thanks!

Reply:
or should i delete those in the PR itself? (below)

Reply:
@TildaDares Done (i deleted those files in the PR, if that's what you meant)

Reply:
Hi @billymoroney1 i think you may need to re-add them based on the main publiclab branch. Here it looks like your PR proposes deleting them from the project. If you just add the exact files as they are in the main branch, it should detect that they are back in sync and no longer appear in the PR. 

Also, I think it's best that you don't include these .gitignore additions, as they're not part of the feature you're working on (our general principle is one "project" per PR). 

Thank you! And no worries, a first PR is a lot of learning curve, it'll go smoothly more and more as you go! Thanks for your time!!! 🎉

Reply:
@jywarren AH i found the public lab github guidelines and forgot to merge my branch back with my fork of the repo before pushing. I'll go try that -

Reply:
(it's such a mess... i might try to backtrack and start clean and do it right)